packages/validation-errors/src/ValidationErrors.tsx

```diff
@@ -59,9 +55,13 @@ function UniquenessError(props: UniquenessErrorProps) {
     [props.localeCode, props.defaultLocaleCode, contentTypesById]
   );
 
+  let conflicting: any[] = []
+  if ('conflicting' in props.error) {
+    conflicting = props.error.conflicting
+  }
   React.useEffect(() => {
     const entryIds = state.entries.map((entry) => entry.id);
-    const conflictIds = props.error.conflicting?.map((entry) => entry.sys.id) || [];
+    const conflictIds = conflicting.map((entry) => entry.sys.id) 
```

Comment:
I'm not sure if this is right - as it doesn't match the type we have in the SDK here https://github.com/contentful/ui-extensions-sdk/blob/0ec6a68cdf04992db31381f13254fe4e2d48d817/lib/types/validation-error.ts#L33

Reply:
The SDK types are probably wrong. I will provide you a link in Slack (public repo :) )